### DISCUSSION TITLE
Fehler bei automatischer Installation

### DISCUSSION THREAD

**Discussion by TimBoWi** [camueller/SmartApplianceEnabler#264] [Installation und Betrieb]
Hallo zusammen,

habe gerade mal aus Langeweile angefangen ein wenig zu basteln. 

Image 2022-01-28-raspios-bullseye-armhf-lite-usbrun mit dem RaspberryPi Imager (Version 1.7.2) auf eine SD-Karte gezogen. Zusätzlich noch einen Bildschirm angeschlossen, damit ich alles verfolgen kann. Die drei Dateien install.config, install.sh und install2.sh habe ich auf einen FAT formatierten USB Stick kopiert (Dateiendungen werden mir auch korrekt angezeigt). 

Karte rein, Strom drauf, und warten bis nur noch die rote LED leuchtet.
Die Rote LED leuchtet bei mir dauerhaft bei folgender Zeile während des Setups und nichts tut sich mehr.

`[OK] Finished "Update UTMP about System Runlevel Changes.`

Im Zweiten Versuch via LAN dann folgendes

`[OK] Finished Daily man-db regeneration`

und wieder... keinen Mucks...

Dann stecke ich laut Anleitung den vorbereiten Stick, der nur die drei Dateien beinhaltet, in einen USB-Port (es leuchtet nur noch die rote LED dauerhaft) aber es geht nicht weiter. Der Stick ist mit FAT Formatiert, keine Änderungen an der install.config vorgenommen, da der Pi via LAN im Netzwerk hängt. USB Stick wird warm. Auch nach 5 Minuten keine Änderung.

Die Rote LED bleibt an, nichts passiert.

Mit WinSCP sehe ich in dem Ordner tmp/Install.log keinerlei Log Datei.
Handelt sich um einen Pi Modell 4. 

Any Ideas?

Würde sonst die Tage mal die manuelle Installation anstoßen, allerdings wundert es mich und ich will den Fehler finden 😄

**Comment by TimBoWi** [comment]
Kurzes Update... kann es sein, dass das Rootsystem zu klein ausfällt. Beim "neu aufsetzen" der Karte ist mir aufgefallen, dass die Boot Partition nur ein paar hundert MB groß ist?

**Comment by camueller** [comment]
Du bist nicht der Erste, der meint, dass die automatische Installation nicht funktioniert. Bisher lag es aber zu 99% daran, dass die Anleitung nicht genau befolgt wurde. Mein letzter Test der automatischen Installation liegt erst einige Wochen zurück ...

**Reply by TimBoWi** [reply]
Schließe ich nicht aus, allerdings wundert es mich, dass der Outcome der Installation im CLI zwei mal ein anderer ist. 

Besonders beim installieren des angepassten Image, kann man aus meiner Sicht nicht wirklich viel falsch machen. 

Ich hab auch kein Problem mit der manuellen Installation. Wollte eher nur darauf hinweisen.

**Reply by TimBoWi** [reply]
update.... manuelle Installation ging ohne weiteres.
Wird vermutlich am Stick gelegen haben, bin aber nicht sicher wo der Fehler lag.
